source/Activated.js

```diff
@@ -1,5 +1,6 @@
 import React, { useCallback, useEffect, useRef, useState } from 'react'
 import { bool, func, node, object, string } from 'prop-types'
+import { useWindowSize } from './helpers'
```

Comment:
How do you feel about having this hook in the helpers?

Reply:
fine by me. happy to add things there until it gets unwieldy